pkg/errorutil/ignore.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package errorutil
+
+import (
+	dmysql "github.com/go-sql-driver/mysql"
+	"github.com/pingcap/errors"
+	tddl "github.com/pingcap/tidb/ddl"
+	"github.com/pingcap/tidb/infoschema"
+	"github.com/pingcap/tidb/parser/mysql"
+)
+
+func IsIgnorableMySQLDDLError(err error) bool {
```

Comment:
please keep the comment, so new code-reader can find a reference.

also, I think these errors is not ignorable. see https://github.com/pingcap/dm/pull/1585:

- ErrDatabaseNotExists
- ErrTableNotExists
- ErrColumnNotExists
- ErrKeyNotExists

cc @amyangfei

Reply:
In TiCDC the DDLs are allowed to be executed reentrant. Seems it is different from DM.

Reply:
ErrDatabaseNotExists is not caused by re-execute DROP DATABASE (that's ErrDatabaseDropExists). more example could be found in https://github.com/pingcap/dm/pull/1585#issuecomment-818383869

ErrDatabaseNotExists is caused by CRAETE TABLE db.tbl while db doesn't exist. I think we should raise error.

Reply:
ok，i will check the example

Reply:
It will be ok if TiCDC integration tests can pass, since there is a ddl reentrant test.

Reply:
Hi @fatelei this comment is not solved, I expected these comments should be added

```
	// we can get error code from:
	// infoschema's error definition: https://github.com/pingcap/tidb/blob/master/infoschema/infoschema.go
	// DDL's error definition: https://github.com/pingcap/tidb/blob/master/ddl/ddl.go
	// tidb/mysql error code definition: https://github.com/pingcap/tidb/blob/master/mysql/errcode.go
```